contrib/python-bindings/source/tria_accessor_wrapper.cc

```diff
@@ -400,6 +408,45 @@ namespace python
       return internal::measure<2, 3, 3>(tria_accessor);
   }
 
+
+
+  unsigned int
+  TriaAccessorWrapper::n_vertices() const
+  {
+    if ((dim == 2) && (spacedim == 2) && (structdim == 1))
+      return internal::tria_accessor_cast<1, 2, 2>(tria_accessor)->n_vertices();
+    else if ((dim == 2) && (spacedim == 3) && (structdim == 1))
+      return internal::tria_accessor_cast<1, 2, 3>(tria_accessor)->n_vertices();
+    else
+      return internal::tria_accessor_cast<2, 3, 3>(tria_accessor)->n_vertices();
```

Comment:
Why don't we have `internal::tria_accessor_cast<1, 3, 3>(tria_accessor)->n_vertices();` and `internal::tria_accessor_cast<3, 3, 3>(tria_accessor)->n_vertices();`?